### PR TITLE
docstring and comments

### PR DESCRIPTION
**Pull request by anas14680**

### CODE REVIEW

File: python_basics.py [hunk 1]
```diff
@@ -1,49 +1,53 @@
...
+            integers.append(l)  # append the list to integers list
         file.close()
-    lst1 = [int(i) for i in integers[0]]
-    lst2 = [int(i) for i in integers[1]]
-    return lst1 + lst2
+
+    return integers
 
 
 def average(lst: list[int]) -> int:
```
Comment (sdonatileach) [inline python_basics.py:21]: we still need to update these to floats rather than integers

File: python_basics.py [hunk 2]
```diff
@@ -79,4 +86,4 @@ def analyze_data(integers, operation="average")  -> int:
         print("The correlation is", ans)
 
     else:
-        return "Enter the right operation value"
+        return "Enter the right operation value"  # we handle exceptions in regards to the operations performed here
```
Comment (sdonatileach) [inline python_basics.py:89]: nice job on all the comments!

Comment (sdonatileach) [inline python_basics.py:89]: nice job on all the comments!

Comment (anas14680) [reply python_basics.py:89]: Thanks!

Comment (anas14680) [reply python_basics.py:89]: Thanks again!

**Comment by anas14680** [comment]
Type hints for the latter functions need to be corrected

File: python_basics.py (hunk 2 above)
Comment (anas14680) [reply python_basics.py:89]: Thanks!

Comment (anas14680) [reply python_basics.py:89]: Thanks again!

**Pull request closed without merging**